build.gradle.kts

```diff
@@ -26,7 +26,6 @@ allprojects {
 
     plugins.withId(rootProject.libs.plugins.kotlin.compose.get().pluginId) {
         extensions.configure<ComposeCompilerGradlePluginExtension> {
-            enableStrongSkippingMode = true
             stabilityConfigurationFile = rootProject.layout.projectDirectory.file("stability_config.conf")
```

Comment:
maybe we don't need stability config now

Reply:
We can.

Reply:
I tried, but we still need it to mark Path and Collections stable.